core/src/edu/cornell/gdiac/main/GameController.java

```diff
@@ -362,6 +399,15 @@ public void draw(float delta) {
 
     level.draw(canvas);
 
+    // Display meter
+    if (!complete && !failed) {
+      displayFont.setColor(Color.BLACK);
+      canvas.begin();
+      String message = "Meter: " + (int) metercounter;
```

Comment:
Can you add some indicator to the timer text for when the game is paused? It could be as simple as prepending `P` to the message string when the game is paused.